lib/model-access-wrapper.js

```diff
@@ -607,7 +607,22 @@ class ModelAccessWrapper extends CrispHooks {
 	_checkDiffPermissions(permissions, doc, data) {
 		const grant = permissions.getTargetGrant(this.options.permissionsTarget, data);
 		const patch = Update.createFromDiff(doc.data, data);
-		const fields = _.pull(createUpdate(patch).getUpdatedFields(), '_id');
+		const schema = this.model.getSchema();
+		let fields = createUpdate(patch).getUpdatedFields();
+		let defaultedFields = [];
+
+		// Find defaulted fields
+		schema.traverseSchema({
+			onSubschema: (subschema, path) => {
+				if (subschema.default !== undefined && getPath(data, path) === undefined) {
+					defaultedFields.push(path);
+				}
+			}
+		});
+
+		// Skip permissions checks on defaulted fields
+		fields = _.pullAll(fields, defaultedFields);
```

Comment:
This will break in cases where a user doesn't have permission to set the default value, but does have permission to set some other value.  Consider the hypothetical scenario where you have a schema intended primarily for internal (server-side) use, with a `status` field similar to the current scenario.  It could have a default value of something like "INTERNAL_READY", which the user cannot set, but possibly allow the user to set it to a value like "CANCELED".

May not be the best example, but there are other similar cases.  Essentially, this breaks the implicitly understood contract that permissions are validated against the document actually going into the database.